app/services/SalesAtVatRateService.scala

```diff
@@ -56,20 +57,6 @@ class SalesAtVatRateService @Inject()() {
     )
   }
 
-  def getTotalNetSales(userAnswers: UserAnswers): BigDecimal   = {
```

Comment:
I assume these weren't used anywhere else anymore?

Reply:
that's right, they weren't used anymore